content/en/docs/concepts/experimental-features.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Experimental features"
+description: "Explanation of experimental features"
+type: docs
+weight: 10
+---
+
+Starting in version `v1.0.0-rc.4`, Notation began introducing experimental features. Experimental features are features that are not yet ready for production use but are available for testing and evaluation. Experimental features may change significantly in later releases or be completely removed. Experimental features are not enabled by default and are not supported.
+
+Currently, Notation has the following experimental features:
+
+- Signing, listing and verifying images as OCI image layout
+- Store signatures using artifact manifest (requires registry support)
```

Comment:
We will soon release RC-5 this week. The second experimental feature will be removed, and a new one will be introduced. I will provide more details once the implementation is done.

Reply:
Thank you @yizha1 

Will also add a link once #196 is merged

Reply:
@zr-msft RC-5 was released, however the blog PR still not merged yet. In RC-5, new flag was introduced named `--allow-referrers-api`, see https://deploy-preview-218--notarydev.netlify.app/blog/2023/announcing-notation-rc5/#change-default-behavior-and-introduce-a-new-flag---allow-referrers-api for details. 

This is the flag that was deprecated in RC-5: https://deploy-preview-218--notarydev.netlify.app/blog/2023/announcing-notation-rc5/#deprecate-flag---signature-manifest-in-notation-sign

Reply:
Thanks @yizha1 amended PR to include your feedback